README.md

```diff
@@ -94,7 +114,24 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
-
+Once a function is defined, we can either call it or reference it. When you call a function, it performs the duties prescribed to it. When you reference a function, it prints out the details of what that function does upon being called.
+
+For example:
+var pizza = {
+  temperature: 70,
+  bake: function() {
+    console.log("Pizza baked to 300 degrees.");
+     this.temperature = 300;
+  }
+};
+pizza.bake(); //CALLING the function
+  performs function and changes temperature from 70 to 300.
+pizza.bake; //REFERENCING the function
+prints out the following:
```

Comment:
Nice example! I think referencing is particular useful in event listeners and call back functions where we don't want to the code to execute immediately until that event happens